### ISSUE TITLE
I can't start KaTrain-v1.1.2

### ISSUE THREAD

**Issue by hope366** [sanderland/katrain#60]
Download KaTrain-v1.1.2, unzip it, and then run KaTrain.exe and The following error occurs
![無題](https://user-images.githubusercontent.com/63999713/83949350-41a5d280-a85e-11ea-9e0f-f8952e16ec63.jpg)
I know this is probably an elementary error, but I'd appreciate any advice.

**Comment by sanderland** [comment]
that looks like your path to the katago config file is wrong - did you change it? what do the general/engine settings look like?

**Comment by hope366** [comment]
Thank you for your response.

>that looks like your path to the katago config file  is wrong - did you change it?

No, I didn't change anything. I just downloaded it, unzipped it and ran KaTrain.exe.

>what do the general/engine settings look like?

The part about "engine" in config.jason looks like this
![無題t](https://user-images.githubusercontent.com/63999713/83954402-db33ab00-a883-11ea-9634-3b7179dd412c.jpg)

**Comment by sanderland** [comment]
That looks ok. Can you set debug level to 1 and copy the output of the console on startup?

**Comment by hope366** [comment]
I don't know how to do "set debug level to 1".
The output of the console at startup is as follows
![無題g](https://user-images.githubusercontent.com/63999713/83954669-32d31600-a886-11ea-8615-99f3170928b4.jpg)

**Comment by hope366** [comment]
Set the debug level to 1 by looking carefully at config.jason I was able to
The output of the console at startup was exactly the same as above.

**Comment by sanderland** [comment]
You probably changed the package config and not your local one. Press F8/General settings, change it there and restart.

**Comment by hope366** [comment]
I pressed F8 and saw KataGo Engine Settings.
In the "Path to KataGo model file" field, I put
'katrain/models/g170e-b15c192-s1672170752-d 466197061.bin.gz
In the "Path to KataGo config file" field, enter
katrain/KataGo/analysis_config.cfg".
I described it as.
I rebooted it and... it worked! Thank you!
It seems that each of the above columns had a different description.

I have BadukMegapack on my PC, with an older version of KaTrain in it. There is. What does this have to do with this time?

Translated with www.DeepL.com/Translator (free version)

**Comment by hope366** [comment]
I pressed F8 and saw KataGo Engine Settings.
In the "Path to KataGo model file" field, I put
'katrain/models/g170e-b15c192-s1672170752-d 466197061.bin.gz
In the "Path to KataGo config file" field, enter
katrain/KataGo/analysis_config.cfg".
I described it as.
I rebooted it and... it worked! Thank you!
It seems that each of the above columns had a different description.

I have BadukMegapack on my PC, with an older version of KaTrain in it. There is. What does this have to do with this time?
